_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+        <table class="attribute-list">
+            <tr>
+                <td>
+                    <strong>Field name</strong>
+                </td>
+                <td>
+                    <strong>Allowed values</strong>
+                </td>
+                <td>
+                    <strong>Allowed special characters</strong>
+                </td>
+            </tr>
+        {% for field in cron.required-fields %}
+            <tr>
+                <td>
+                    {{ field.name }}
+                </td>
+                <td>
+                    {{ field.allowed-values }}
+                </td>
+                <td>
+                    {{ field.special-characters }}
+                </td>
+            </tr>
+        {% endfor %}
+        </table>
+    </div>
+
+    <div role="tabpanel" class="tab-pane" id="special-characters">
+
+        <p>Each field in a cron expression has its own list of allowed special characters. These characters allow you to select all values, a list of values, or a range of values. Fields may also contain both a range and a list. Refer to the <a href="#examples">Example replication schedules section</a> for examples.</p> 
+
+        <p><strong>Note</strong>: Entering anything other than a field's allowed values or special characters will result in an error when you try to save the integration's settings. Refer to the <a href="#troubleshooting-cron-errors">Troubleshooting section</a> for help resolving these errors.</p>
+        <table class="attribute-list">
+            <tr>
+                <td width="15%; fixed" align="right">
+                    <strong>Special<br>character</strong>
+                </td>
+                <td width="15%; fixed">
+                    <strong>Allowed<br>fields</strong>
+                </td>
+                <td>
+                    <strong>Description</strong>
+                </td>
+            </tr>
+        {% for character in cron.special-characters %}
+            <tr>
+                <td align="right">
+                    <strong>{{ character.character }}</strong><br>
+                    ({{ character.name }})
+                </td>
+                <td>
+                    {{ character.allowed-in }}
+                </td>
+                <td class="description">
+                    {{ character.description | markdownify }}
+
+                    <strong>Example:</strong> {{ character.example | markdownify }}
+                </td>
+            </tr>
+        {% endfor %}
+        </table>
+    </div>
+</div>
+<br>
+
+### Specifying days in cron expressions {#specifying-days}
+
+Stitch's implementation of cron uses [POSIX](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html){:target="new"}, which, unlike other cron libraries, uses a unique method to handle the specification of days. This section will focus on the behavior of the `day of month`, `month`, and `day of week` fields.
```

Comment:
I think this section is slightly misleading. It implies that there are cron libraries that don't have this combinatoric behavior when in fact all cron implementation do. It's specifically [Quartz](http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html) which is only cron-like that doesn't have it.

Again, I'm not sure how much of that detail to expose to our customers.

Reply:
This might just be as simple as taking out the `unlike other cron libraries` bit, so it'd read something like:

> Stitch's implementation of cron uses POSIX, which uses a unique method to handle the specification of days.

Reply:
That's possible. Maybe this as well:

> Stitch's cron scheduling implementation is POSIX compliant. The POSIX standard has subtle rules about how the final 3 fields of the cron expression affect the schedule on which the integration will run. This section…

Specifically `POSIX compliant` is how most software talks about whether or not it follows the POSIX specification.

Reply:
Updated to:

> Stitch's implementation of cron is [POSIX](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html) compliant. The POSIX standard has subtle rules about how the `day of month`, `month`, and `day of week` fields affect the schedule on which the integration will run. This section will focus on the behavior of those fields.

Reply:
🙌